aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -114,6 +115,33 @@
    */
   public static final String S3FILEIO_ACL = "s3fileio.acl";
 
+  /**
+   * If {@link org.apache.iceberg.aws.glue.GlueCatalog} should use external lock or not, default to false.
+   * If set to true, it will use DynamoDB to enforce locking during commits.
+   */
+  public static final String GLUE_CATALOG_LOCK_ENABLED = "gluecatalog.lock.enabled";
```

Comment:
I'd prefer not to do this because it breaks `grep` or github search to find configuration strings. I think it is worth the duplication of "gluecatalog" to make it easy to search and find these.